etc/HIReplayOfflineConfiguration.py

```diff
@@ -1346,7 +1382,7 @@
                do_reco=False,
                aod_to_disk=False,
                raw_to_disk=False,
-               scenario=hiTestppScenario)
+               scenario=hiLightEventsScenario)
```

Comment:
@LinaresToine I don't understand this choice (and also several others).
In 2023 this PD was processed with `ppEra_Run3_pp_on_PbPb_2023`. I think this should be `hiHeavyEventsScenario` (= `"ppEra_Run3_pp_on_PbPb_2024"`) now.